ydb/library/yql/parser/pg_wrapper/arrow.cpp

```diff
@@ -204,8 +207,28 @@ std::shared_ptr<arrow::Array> PgDecimal128ConvertNumeric(const std::shared_ptr<a
     return ret;
 }
 
-Numeric PgDecimal128ToNumeric(int64_t high_bits, uint64_t low_bits, int32_t precision, int32_t scale) {
-    Numeric res = int64_div_fast_to_numeric(low_bits, scale);
+Numeric PgDecimal128ToNumeric(arrow::Decimal128 value, int32_t precision, int32_t scale, Numeric& high_bits_mul) {
```

Comment:
Numeric is a pointer
https://doxygen.postgresql.org/numeric_8h.html
pass by value